### DISCUSSION TITLE
Uso de qsort [stdlib]

### DISCUSSION THREAD

**Discussion by boot-kitty** [IIC2133-PUC/2024-2#190] [Tarea 3] [answered]
Hola! Quisiera consultar si al igual que en la entrega anterior está permitido hacer uso de la implementación para el algoritmo quicksort de stdlib ([qsort()](https://www.ibm.com/docs/en/i/7.5?topic=functions-qsort-sort-array)). 

Saludos!

**Comment by Segismundios** [comment, marked as answer]
Holaa @su-hoxton. Sí está permitido el uso de qsort de stdlib.
Saludos!

**Reply by boot-kitty** [reply]
Muchas gracias!
